client/stylesheets/_responsive.scss

```diff
@@ -479,6 +493,21 @@
     }
 }
 
+@media (max-width: 375px) {
+    .profileCardInfo li:first-child {
```

Comment:
Line should be indented 2 spaces, but was indented 4 spaces<br>Selector `profileCardInfo` should be written in lowercase with hyphens